examples/dicklang/parse_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParser(t *testing.T) {
+	var src = `
+	B===> 8==>  B::>
+		<====8 <::::::3
+			8xxxx> 8xxx=xxx>
+		B:x:=:x>
+	 <:=3
+	`
+	mod, err := Parse("sample.dicklang", []rune(src))
+	require.NoError(t, err)
+	require.Len(t, mod.Dicks, 9)
+	require.Equal(t, mod.Dicks[0].Frag.Src(), []rune("B===>"))
```

Comment:
Oh, I just noticed, you got the order of the arguments wrong, the expected value comes first, see here: https://godoc.org/github.com/stretchr/testify/require#Equal